packages/boxel/addon/components/boxel/org-header/usage.hbs

```diff
@@ -1,11 +1,11 @@
 <Freestyle::Usage @name="Boxel:OrgHeader">
   <:description>
-    Usually shown at the top of a dashboard
+    Usually shown at the top of a dashboard. See also Boxel::OrgTitle
```

Comment:
Would it make sense to have a similar “See also” on `org-title`? Also, can this be a link?

Reply:
Yes, will do both. Thanks!